python/vrp.py

```diff
@@ -0,0 +1,82 @@
+from mage.geography import create_distance_matrix, LATITUDE, LONGITUDE
+from mage.constraint_programming import VRPConstraintProgrammingSolver
+from typing import Dict, List
+
+import mgp
+
+
+__distance_matrix = None
+__depot_index = None
+
+
+def get_distance_matrix(vertices):
+    """
+    Assigns distance matrix global object or returns if its already there.
+    """
+    global __distance_matrix
+
+    if __distance_matrix is not None:
+        return __distance_matrix
+
+    vertex_positions: List[Dict[str, float]] = []
+    for vertex in vertices:
+        vertex_positions.append(
+            {LATITUDE: vertex.properties.get(LATITUDE), LONGITUDE: vertex.properties.get(LONGITUDE)}
+        )
+
+    __distance_matrix = create_distance_matrix(vertex_positions)
+
+    return __distance_matrix
+
+
+def get_depot_index(vertices, depot_label):
+    """
+    Assigns depot index global variable or returns if its already there.
+    """
+    global __depot_index
+
+    if __depot_index is not None:
+        return __depot_index
+
+    for i, vertex in enumerate(vertices):
+        if depot_label in vertex.labels:
+            __depot_index = vertices[i].id
+            break
```

Comment:
```suggestion
    for vertex in vertices:
        if depot_label in vertex.labels:
            __depot_index = vertex.id
            break
```

Reply:
Thanks, I actually meant to write `__depot_index = i` but since I deleted everything from the database, id got equal to index